configure.ac

```diff
@@ -1,11 +1,11 @@
 dnl required version of autoconf
 AC_PREREQ([2.53])
 
-AC_INIT([libgstspotify],[0.1.0])
+AC_INIT([libgstspotify],[0.2.0])
```

Comment:
What do you think about bumping this to 1.0.0? It's a very major change to it and it's definitely not compatible with the 0.1.0 version.